crates/next-core/js/src/entry/next-hydrate.tsx

```diff
@@ -20,6 +36,13 @@ import * as page from ".";
     assetPrefix,
   });
 
+  // for the page loader
+  window.__turbopack_load_page_chunks__ = (page, paths) => {
+    Promise.all(paths.map((p) => loadPageChunk(assetPrefix, p))).catch((err) =>
```

Comment:
```suggestion
    Promise.all(paths.map((p) => __turbopack_load__(assetPrefix + p))).catch((err) =>
```

We already have a method to load JS or CSS chunks in the runtime code. You can use that one.

Reply:
Subscribing to new chunks is needed as `subscribeToCssChunkUpdates` above. Probably `__turbopack_load__` should do that as that instead, so `import()` will work too.

Reply:
Not sure how to get `__turbopack_load__ ` to do that right now, changed it to use `__turbopack_load__` though and added a TODO for later and created a ticket